api/ResizeObserver.json

```
@@ -29,7 +29,7 @@
             "version_added": "47"
           },
           "safari": {
-            "version_added": false
+            "version_added": "13.1"
           },
           "safari_ios": {
             "version_added": false
```

It looks like these changes have also shipped to Safari for iOS. Would you mind setting the iOS values to `"13.4"`, as well? Like this:

```suggestion
            "version_added": "13.4"
```

---

Done.